readme/configuration-management.md

```diff
@@ -143,4 +178,6 @@ If you need to override the default configuration provided by another project (o
 
 ### Other gotchas
 
+Be aware that reverting all features and config on every deploy creates a risk of discarding server-side changes. This risk should be controlled by carefully managing permissions, and must be balanced against the greater risk of allowing for divergent configuration between your DB and VCS.
+
```

Comment:
Maybe add one more gotcha that Configuration Management in Drupal 8 is still being improved early in the Drupal 8 lifecycle, and you should continue to monitor Drupal Core's issue queue and Drupal Planet blog posts for refinements to the CM workflows explained here.